packages/annotations-compiler/compiler/src/main/java/com/facebook/react/module/processing/BUCK

```diff
@@ -0,0 +1,20 @@
+load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep", "react_native_target", "rn_java_library")
+
+rn_java_library(
+    name = "processing-lib",
```

Comment:
Hmm not sure what it could be, this target was moved from https://github.com/facebook/react-native/blob/main/ReactAndroid/src/main/java/com/facebook/react/module/processing/BUCK#L15 but should not have changed